src/HotChocolate/Core/src/Types/Utilities/ITypeConverter.cs

```
@@ -2,7 +2,13 @@ namespace HotChocolate.Utilities;
 
 public interface ITypeConverter
 {
-    bool TryConvert(Type from, Type to, object? source, out object? converted);
+    bool TryConvert(Type from, Type to, object? source, out object? converted, out Exception? conversionException)
```

I would get rid of the default implementation. It should be a really rare case that somehow has overridden this and the default just makes it really easy to forgot to forward the exception as expected.

---

Done. I thought about this too, but wasn’t sure if a source-breaking change is okay here. I’ll mention it in the breaking-changes doc PR.

Your opinion on the `TypeConverterExtensions`? I left them as is, but ofc they’re right now swallowing the exception.